study-builder/studies/mbc/subs.conf

```diff
@@ -0,0 +1,22 @@
+{
+  "i18n": {
+    "en": {include required("i18n/en.conf")
+           include required("../snippets/i18n/en.conf")},
+    "es": {include required("i18n/es.conf")
+           include required("../snippets/i18n/es.conf")},
+  }
```

Comment:
This is where the translations are linked back to the source files.
This setup depends on using the `--substitutions subs.conf` in the study builder command line.